tests/test_finders.py

```diff
@@ -60,44 +60,57 @@ def do_collect():
 
 common_settings = {
     "STATIC_URL": "static/",
-    "STATICFILES_DIRS": [Path(__file__).resolve().parent / "components"],
     "STATIC_ROOT": "staticfiles",
     "ROOT_URLCONF": __name__,
     "SECRET_KEY": "secret",
+    "COMPONENTS": {
+        "dirs": [Path(__file__).resolve().parent / "components"],
+    },
+    "INSTALLED_APPS": ("django_components", "django.contrib.staticfiles"),
 }
 
 
-# Check that .py and .html files are INCLUDED with the original staticfiles app
+# Check that the component files are NOT loaded when our finder is NOT added
 @override_settings(
     **common_settings,
-    INSTALLED_APPS=("django_components", "django.contrib.staticfiles"),
+    STATICFILES_FINDERS=[
```

Comment:
I reused the tests for `safer_staticfiles` to check that we indeed load only JS and CSS